Cargo.lock

```diff
@@ -171,9 +171,9 @@ checksum = "5827cebf4670468b8772dd191856768aedcb1b0278a04f989f7766351917b9dc"
 
 [[package]]
 name = "cpufeatures"
-version = "0.2.3"
+version = "0.2.4"
```

Comment:
let's remove changes in this file

Reply:
its automatically genearating  version = "0.2.4" when running the bot